src/components-v2/ExperienceList/ExperienceList.tsx

```diff
@@ -37,6 +37,13 @@ const ExperienceList: React.FC<ExperienceListProps> = ({
     else setSelectable(false);
   }, []);
 
+  useEffect(() => {
+    if (router.query?.type === 'experience') {
```

Comment:
please add some comment like
// TODO: make `experience` as global constant for helper query instead hardcoded